src/Tribe/PUE/Checker.php

```diff
@@ -467,68 +467,50 @@ public function ajax_validate_key() {
 		/**
 		 * processes variable substitutions for server-side API message
 		 */
-		private function get_api_message( $info ) {
+		private function get_api_message( $pluginInfo ) {
 			// this default message should never show, but is here as a fallback just in case.
 			$message = sprintf(
 				esc_html__( 'Sorry, there is a problem with your license key. You\'ll need to %scheck your license%s to have access to updates, downloads, and support.', 'tribe-common' ),
 				'<a href="https://theeventscalendar.com/license-keys/">',
 				'</a>'
 			);
 
-			if ( ! empty( $info->api_invalid_message ) ) {
-				$message = wp_kses( $info->api_invalid_message, 'post' );
+			if ( ! empty( $pluginInfo->api_invalid_message ) ) {
+				$message = wp_kses( $pluginInfo->api_invalid_message, 'post' );
 			}
 
 			$message = str_replace( '%plugin_name%', '<b>' . $this->get_plugin_name() . '</b>', $message );
 			$message = str_replace( '%plugin_slug%', $this->get_slug(), $message );
 			$message = str_replace( '%update_url%', $this->get_pue_update_url(), $message );
-			$message = str_replace( '%version%', $info->version, $message );
+			$message = str_replace( '%version%', $pluginInfo->version, $message );
 
 			return $message;
 		}
 
 		/**
-		 * Echo JSON formatted errors
+		 * Displays an error notice if a premium plugin is activated and the license is expired
+		 *
+		 * @since 4.3
 		 */
-		public function display_json_error() {
```

Comment:
As we discussed the other day on our call, I don't think we want to lose this method or the message it provides. This message notifies the user when the API key that they entered is invalid.